packages/manager/apps/telecom/src/app/telecom/telephony/service/fax/customDomains/custom-domains.html

```diff
@@ -45,23 +45,18 @@
                         >
                             <span data-ng-bind="$row.domain"></span>
                         </oui-datagrid-column>
-                        <oui-datagrid-column
-                            data-title="'telephony_service_fax_custom_domains_configuration_table_column_action' | translate"
+                        <oui-action-menu
+                            aria-label="{{:: 'telephony_service_fax_custom_domains_configuration_table_column_action' | translate }}"
+                            compact
+                            data-placement="end"
                         >
-                            <button
-                                type="button"
-                                class="btn btn-default"
-                                data-ng-click="$ctrl.removeCustomDomains($row)"
+                            <oui-action-menu-item
+                                on-click="$ctrl.removeCustomDomains($row)"
                                 data-ng-disabled="$row.isDeleting"
-                                data-translate-attr="{ title: 'telephony_service_fax_custom_domains_configuration_table_action_delete' }"
```

Comment:
**suggestion:** Looks like this translation is no more used. Maybe we can safely remove it from here: https://github.com/ovh/manager/blob/8ff607673c56e34a530deea57b2d1914b64581e3/packages/manager/apps/telecom/src/app/telecom/telephony/service/fax/customDomains/translations/Messages_fr_FR.json#L8